src/ProductImportRulesChecker/ProductImportRulesChecker.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\ProductImportRulesChecker;
+
+
+use http\Exception\BadMethodCallException;
+
+/**
+ * This class check import rules to each product from csv file.
+ */
+class ProductImportRulesChecker
+{
+
+    private string $errorMessage;
+    private const COST="Cost in GBP";
+    private const STOCK="Stock";
+
+    public function __construct()
+    {
+        $this->errorMessage="";
+    }
+
+    public function check(array $product):bool
+    {
+        if($this->checkFirstRule($product) && $this->checkSecondRule($product) && $this->checkThirdRule($product)){
```

Comment:
unacceptable metods naming. unacceptable formatting.
You validator can be done like this.
```
public function validate (array $product): void
{
    $this->validateProductCost($product);
    $this->validateProductStockCount($product);
}
```
and throw exception like InvalidProductDataException with exception message like "Product cost is less than 5 GBP" or any other